setup.cfg

```diff
@@ -0,0 +1,6 @@
+[bdist_rpm]
+vendor="British Crown (Met Office) & Contributors"
+packager="isodatetime devs"
+
+[install]
+install-lib=/usr/lib/python2.7/site-packages
```

Comment:
Is Travis CI failing because of this line?

Reply:
Oh! You might have found the error! This would change the installation location for the `pip` package too... quite interesting. Will try to fix it tomorrow. Thanks for the great spot!

Reply:
So yesterday after reading your comment at home, I thought... well, gotta have to maintain a `spec` file for the RPM... but really don't want to introduce that right now as we have other pending issues (python3, web gui, etc).

This morning, I spent 15 minutes just looking at the code and blinking only the necessary amount :eyes: then realized that 

- if it's just the `name` parameter that we need to change, we could override it someway programmatically in `setup.py` (which is common, look at any medium/large project's `setup.py`). Found a good StackOverflow example that worked like a charm.

- the `install_lib` had to be specified for me, as I am building under Ubuntu. Then, as we got #111 merged, and I started using Anaconda, I realized it used `/home/kinow/.../anaconda/usr/lib/....`... which means that directory depends on the machine building the RPM... So I reckon it's fair to leave that setting as is, without any customization. And let dev/users build the RPM in CentOS, or even use [Travis CI hacks to build with Docker+CentOS](https://derekweitzel.com/2016/05/03/building-centos-packages-on-travisci/).

Updated pull request to have the custom `name` for `bdist_rpm`. To test, one can simply try:

- `python setup.py bdist`, which will produce the default binary distribution, i.e. a `.tar.gz` file
- `python setup.py bdist_rpm`, which will produce the RPM file and RPM source file

If anyone would like to test even further, just grab the RPM, and - in a system without `isodatetime`, try running `rpm -ivh python-isodatetime*noarch.rpm`, check the output, and the try importing `isodatetime` (e.g. `python -c 'import isodatetime'`).